inst/makeData.R

```diff
@@ -83,5 +83,29 @@ d = load2("../thirdparty/yeast.RData")
 yeast.task = makeMultilabelTask("yeast-example", data = d, target = paste0("label", 1:14))
 save(yeast.task, file = file.path(dn, "yeast.task.RData"), compress = "xz")
 
+# FDA classification
+set.seed(DATASEED)
+gunpoint = load2("../thirdparty/gunpoint.RData")
+gunpoint.task = makeFDAClassifTask(data = gunpoint, target = "X1", positive = "1")
+save(gunpoint.task, file = file.path(dn, "gunpoint.task.RData"), compress = "xz")
 
-
+# FDA regression
+set.seed(DATASEED)
+data(fuelSubset, package = "FDboost")
+fuelSubset$UVVIS = scale(fuelSubset$UVVIS, scale = FALSE)
+fuelSubset$NIR = scale(fuelSubset$NIR, scale = FALSE)
+fuelSubset$uvvis.lambda = with(fuelSubset, (uvvis.lambda - min(uvvis.lambda)) /
+                                  (max(uvvis.lambda) - min(uvvis.lambda) ))
+fuelSubset$nir.lambda = with(fuelSubset, (nir.lambda - min(nir.lambda)) /
+                                (max(nir.lambda) - min(nir.lambda) ))
+len1 = length(fuelSubset$uvvis.lambda)
+len2 = length(fuelSubset$nir.lambda)
+
+# mdata = data.frame(fuelSubset[c("UVVIS", "NIR", "h2o", "heatan")]) # Potential error, not doing it now. Cleared on friday.
+mdata = as.data.frame(Reduce(cbind, list(fuelSubset$UVVIS, fuelSubset$NIR,
```

Comment:
The problem is if we just use cbind() or even just data.frame, they are potential risk, which I discussed with Fabian Scheipl, he would suggest we don't do this. We could have it now, and discuss with Fabian on Friday.  @giuseppec